index.js

```diff
@@ -86,20 +77,23 @@ async function getPronouns(userid) {
 }
 
 let lastMessage = "";
-let enableLocal = false;
+let initialHistory = [
```

Comment:
normalise using const with a constant!! (correct me if i'm wrong but i don't see any mutations of initialHistory, always use a const by default. just small nitpicks)